automatic/pencil/legal/LICENSE.md.txt

```diff
@@ -0,0 +1,339 @@
+		    GNU GENERAL PUBLIC LICENSE
```

Comment:
You can leave the filename of this LICENSE as `LICENSE.md` (or `LICENSE.txt`).
Doens't really matter though, so feel free to ignore this one.